erts/mac/man/man3/CosNotification.3
.TH CosNotification 3 "cosNotification 1.2" "Ericsson AB" "Erlang Module Definition"
.SH NAME
CosNotification \- This module export functions which return QoS and Admin Properties constants.
.SH DESCRIPTION
.LP
To get access to all definitions include necessary \fIhrl\fR\& files by using: 
.br
\fI-include_lib("cosNotification/include/*\&.hrl")\&.\fR\&
.SH EXPORTS
.LP
.B
\&'EventReliability\&'() -> string()
.br
.RS
.LP
This function returns the EventReliability QoS identifier
.RE
.LP
.B
\&'BestEffort\&'() -> short()
.br
.RS
.LP
This function returns the BestEffort QoS value\&.
.RE
.LP
.B
\&'Persistent\&'() -> short()
.br
.RS
.LP
This function returns the Persistent QoS value\&.
.RE
.LP
.B
\&'ConnectionReliability\&'() -> string()
.br
.RS
.LP
This function returns the ConnectionReliability QoS identifier\&.
.RE
.LP
.B
\&'Priority\&'() -> string()
.br
.RS
.LP
This function returns the Priority QoS identifier\&.
.RE
.LP
.B
\&'LowestPriority\&'() -> short()
.br
.RS
.LP
This function returns the LowestPriority QoS value\&.
.RE
.LP
.B
\&'HighestPriority\&'() -> short()
.br
.RS
.LP
This function returns the HighestPriority QoS value\&.
.RE
.LP
.B
\&'DefaultPriority\&'() -> short()
.br
.RS
.LP
This function returns the DefaultPriority QoS value\&.
.RE
.LP
.B
\&'StartTime\&'() -> string()
.br
.RS
.LP
This function returns the StartTime QoS identifier\&.
.RE
.LP
.B
\&'StopTime\&'() -> string()
.br
.RS
.LP
This function returns the StopTime QoS identifier\&.
.RE
.LP
.B
\&'Timeout\&'() -> string()
.br
.RS
.LP
This function returns the Timeout QoS identifier\&.
.RE
.LP
.B
\&'OrderPolicy\&'() -> string()
.br
.RS
.LP
This function returns the OrderPolicy QoS identifier\&.
.RE
.LP
.B
\&'AnyOrder\&'() -> short()
.br
.RS
.LP
This function returns the AnyOrder QoS value\&.
.RE
.LP
.B
\&'FifoOrder\&'() -> short()
.br
.RS
.LP
This function returns the FifoOrder QoS value\&.
.RE
.LP
.B
\&'PriorityOrder\&'() -> short()
.br
.RS
.LP
This function returns the PriorityOrder QoS value\&.
.RE
.LP
.B
\&'DeadlineOrder\&'() -> short()
.br
.RS
.LP
This function returns the DeadlineOrder QoS value\&.
.RE
.LP
.B
\&'DiscardPolicy\&'() -> string()
.br
.RS
.LP
This function returns the DiscardPolicy QoS identifier\&.
.RE
.LP
.B
\&'LifoOrder\&'() -> short()
.br
.RS
.LP
This function returns the LifoOrder QoS value\&.
.RE
.LP
.B
\&'RejectNewEvents\&'() -> short()
.br
.RS
.LP
This function returns the RejectNewEvents QoS value\&.
.RE
.LP
.B
\&'MaximumBatchSize\&'() -> string()
.br
.RS
.LP
This function returns the MaximumBatchSize QoS identifier\&.
.RE
.LP
.B
\&'PacingInterval\&'() -> string()
.br
.RS
.LP
This function returns the PacingInterval QoS identifier\&.
.RE
.LP
.B
\&'StartTimeSupported\&'() -> string()
.br
.RS
.LP
This function returns the StartTimeSupported QoS identifier\&.
.RE
.LP
.B
\&'StopTimeSupported\&'() -> string()
.br
.RS
.LP
This function returns the StopTimeSupported QoS identifier\&.
.RE
.LP
.B
\&'MaxEventsPerConsumer\&'() -> string()
.br
.RS
.LP
This function returns the MaxEventsPerConsumer QoS identifier\&.
.RE
.LP
.B
\&'MaxQueueLength\&'() -> string()
.br
.RS
.LP
This function returns the MaxQueueLength Admin identifier\&.
.RE
.LP
.B
\&'MaxConsumers\&'() -> string()
.br
.RS
.LP
This function returns the MaxConsumers Admin identifier\&.
.RE
.LP
.B
\&'MaxSuppliers\&'() -> string()
.br
.RS
.LP
This function returns the MaxSuppliers Admin identifier\&.
.RE

erts/mac/lib/cosNotification-1.2/src/CosNotification.erl
%%  coding: latin-1
%%------------------------------------------------------------
%%
%% Implementation stub file
%% 
%% Target: CosNotification
%% Source: /private/tmp/erlang20150820-34163-1k9di6g/otp-OTP-18.0.3/lib/cosNotification/src/CosNotification.idl
%% IC vsn: 4.4
%% 
%% This file is automatically generated. DO NOT EDIT IT.
%%
%%------------------------------------------------------------

-module('CosNotification').
-ic_compiled("4_4").


%% Interface functions
-export(['EventReliability'/0, 'BestEffort'/0, 'Persistent'/0]).
-export(['ConnectionReliability'/0, 'Priority'/0, 'LowestPriority'/0]).
-export(['HighestPriority'/0, 'DefaultPriority'/0, 'StartTime'/0]).
-export(['StopTime'/0, 'Timeout'/0, 'OrderPolicy'/0]).
-export(['AnyOrder'/0, 'FifoOrder'/0, 'PriorityOrder'/0]).
-export(['DeadlineOrder'/0, 'DiscardPolicy'/0, 'LifoOrder'/0]).
-export(['RejectNewEvents'/0, 'MaximumBatchSize'/0, 'PacingInterval'/0]).
-export(['StartTimeSupported'/0, 'StopTimeSupported'/0, 'MaxEventsPerConsumer'/0]).
-export(['MaxQueueLength'/0, 'MaxConsumers'/0, 'MaxSuppliers'/0]).

%%%% Constant: 'EventReliability'
%%
'EventReliability'() -> "EventReliability".

%%%% Constant: 'BestEffort'
%%
'BestEffort'() -> 0.

%%%% Constant: 'Persistent'
%%
'Persistent'() -> 1.

%%%% Constant: 'ConnectionReliability'
%%
'ConnectionReliability'() -> "ConnectionReliability".

%%%% Constant: 'Priority'
%%
'Priority'() -> "Priority".

%%%% Constant: 'LowestPriority'
%%
'LowestPriority'() -> -32767.

%%%% Constant: 'HighestPriority'
%%
'HighestPriority'() -> 32767.

%%%% Constant: 'DefaultPriority'
%%
'DefaultPriority'() -> 0.

%%%% Constant: 'StartTime'
%%
'StartTime'() -> "StartTime".

%%%% Constant: 'StopTime'
%%
'StopTime'() -> "StopTime".

%%%% Constant: 'Timeout'
%%
'Timeout'() -> "Timeout".

%%%% Constant: 'OrderPolicy'
%%
'OrderPolicy'() -> "OrderPolicy".

%%%% Constant: 'AnyOrder'
%%
'AnyOrder'() -> 0.

%%%% Constant: 'FifoOrder'
%%
'FifoOrder'() -> 1.

%%%% Constant: 'PriorityOrder'
%%
'PriorityOrder'() -> 2.

%%%% Constant: 'DeadlineOrder'
%%
'DeadlineOrder'() -> 3.

%%%% Constant: 'DiscardPolicy'
%%
'DiscardPolicy'() -> "DiscardPolicy".

%%%% Constant: 'LifoOrder'
%%
'LifoOrder'() -> 4.

%%%% Constant: 'RejectNewEvents'
%%
'RejectNewEvents'() -> 5.

%%%% Constant: 'MaximumBatchSize'
%%
'MaximumBatchSize'() -> "MaximumBatchSize".

%%%% Constant: 'PacingInterval'
%%
'PacingInterval'() -> "PacingInterval".

%%%% Constant: 'StartTimeSupported'
%%
'StartTimeSupported'() -> "StartTimeSupported".

%%%% Constant: 'StopTimeSupported'
%%
'StopTimeSupported'() -> "StopTimeSupported".

%%%% Constant: 'MaxEventsPerConsumer'
%%
'MaxEventsPerConsumer'() -> "MaxEventsPerConsumer".

%%%% Constant: 'MaxQueueLength'
%%
'MaxQueueLength'() -> "MaxQueueLength".

%%%% Constant: 'MaxConsumers'
%%
'MaxConsumers'() -> "MaxConsumers".

%%%% Constant: 'MaxSuppliers'
%%
'MaxSuppliers'() -> "MaxSuppliers".

